src/engine/effect_functions.asm

```diff
@@ -143,7 +146,8 @@ Func_2c0bd: ; 2c0bd (b:40bd)
 	call ShuffleDeck
 	ret
 
-Func_2c0c7: ; 2c0c7 (b:40c7)
+; return carry if Turn Duelist is the Player
+CheckIfTurnDuelistIsPlayer: ; 2c0c7 (b:40c7)
```

Comment:
Given that DUELIST_TYPE_PLAYER matches everything that's not an opponent's turn  how generic this function seems to be, I wonder if it could be just renamed to IsPlayerTurn.